dmscripts/supplier_data.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""Helper classes for fetching supplier data given a client."""
+from collections import OrderedDict
+from datetime import date
+from itertools import groupby
+
+from dmutils.formats import DISPLAY_DATE_FORMAT
+
+
+class SupplierFrameworkData(object):
+    """Class to get supplier data from the dmapiclient."""
+
+    data = None
+
+    def __init__(self, client, target_framework_slug):
+        self.client = client
+        self.target_framework_slug = target_framework_slug
+
+    def get_supplier_frameworks(self):
+        """Return supplier frameworks."""
+        return self.client.find_framework_suppliers(self.target_framework_slug)['supplierFrameworks']
+
+    def get_supplier_users(self):
+        """Return a dict, supplier id: application status."""
```

Comment:
don't think this docstring is accurate?  Isn't it `supplier_id: [list of supplier users]`?